### PR TITLE
USE_FAST_NVCC Windows

### PR DESCRIPTION
**Pull request by mantaionut**
USE_FAST_NVCC now works on Windows.

Fixes #67100

### CODE REVIEW

File: tools/fast_nvcc/wrap_nvcc.bat.in [hunk 1]
```diff
@@ -0,0 +1 @@
+python "@FAST_NVCC_EXECUTABLE@" --nvcc "@CUDA_NVCC_EXECUTABLE_ORIGIN@" -- %*
```
Comment (ezyang) [inline tools/fast_nvcc/wrap_nvcc.bat.in:1]: trailing newline missing

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
you didn't give maintainers permission to rebase your PR, you will have to do it manually

File: tools/fast_nvcc/wrap_nvcc.bat.in (hunk 1 above)
Comment (mantaionut) [reply tools/fast_nvcc/wrap_nvcc.bat.in]: Made the change

**Comment by mantaionut** [comment]
@ezyang made the rebase and your requested newline change.

**Comment by malfet** [comment]
It would be nice to have some tests, though `fast_nvcc` is mostly moot, since `--threads` option was added to NVCC in CUDA-11.4

**Pull request closed without merging**